### PR TITLE
Update templates 

### PR DESCRIPTION
**Pull request by ryekerjh**
## Changes
1. Adds docs for Github's new workflow for adding issue templates
2. Adds docs to true up the pull request template so that it is picked up automatically by github

## Approach
Referencing Github's new docs on the topic. 

## Learning
[Github Template Docs](https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-_This goes in your project's root directory under `.github/PULL_REQUEST_TEMPLATE/` folder_
```
Comment (kmausolf) [inline .github/PULL_REQUEST_TEMPLATE.md]: Thank you!  I have to delete this every time I open a PR.

File: standards/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,32 +1,16 @@
...
-_Links to blog posts, patterns, libraries or addons that could be used to solve this problem_
+#### To add issue templates to your repository:
 
+1. Go to your repository's `Settings` tab
+2. Under the `Options` section, scroll down to **Features**
+3. Make sure **Issues** option is selected (check the box) and click the `Set up templates` button
+4. Select which type of template you want to set up. Github has pre-configured templates that you can choose to use. You can edit the default templates if you need to, or you can add a custom template manually
+5. Once you are happy with the template, click on the green `Propose changes` button at the top right of the page
+6. Fill out the commit information that pops up. Choose the option to create a new branch and start a pull request rather than committing directly to your default branch
+7. Have your team review and approve your pull request. Once it is closed, you will have issue templates
```
Comment (kmausolf) [inline standards/ISSUE_TEMPLATE.md]: This must be pretty recent, but that's a hot tip.

Comment (ryekerjh) [reply standards/ISSUE_TEMPLATE.md]: VERY recent! Hooray!

**Comment by Basim-Elayan** [comment]
Looks awesome for me.

File: standards/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,4 +1,19 @@
-_This goes in your project's root directory under `.github/` folder_
+## Pull Request Templates
+
+[Github documentation](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository#adding-a-pull-request-template)
+#### To add an issue template to your repository:
+
+1. Create a file named `PULL_REQUEST_TEMPLATE.md` inside your **.github** folder
+2. Put the markdown that you want for your Pull Request template in this file. Below is an example of one potential template. Feel free to use it for your repository or create your own.
+3. Open a pull request to your default branch. This will only work if the exact file structure and file name referenced in step 1 is in your default branch.
+4. Have your team review and approve your pull request. Once it is closed, you will have pull request templates
```
Comment (carlosvargas) [inline standards/PULL_REQUEST_TEMPLATE.md]: Should we mention that you can tag `PR` team for code reviews? or is that already here and I missed it?

Comment (coreyshuman) [reply standards/PULL_REQUEST_TEMPLATE.md]: Agreed, I think we should mention including the PR Team in both cases. Other than that, this has my seal of approval!

Comment (mwallert) [reply standards/PULL_REQUEST_TEMPLATE.md]: Agreed

**Pull request merged**